packages/lit-html/package.json

```diff
@@ -13,9 +13,13 @@
       "development": "./development/lit-html.js",
       "default": "./lit-html.js"
     },
-    "./parts.js": {
-      "development": "./development/parts.js",
-      "default": "./parts.js"
+    "./directive.js": {
+      "development": "./development/directive.js",
+      "default": "./directive.js"
+    },
+    "./directive-helpers.js": {
+      "development": "./development/directive-helpers.js",
+      "default": "./directive-helpers.js"
```

Comment:
oops, I didn't add `disconnectable-directive.js` to the exports; could you do that?

Reply:
done